### PR TITLE
AP/IB page

### PR DESCRIPTION
**Pull request by oo-ng**
#### What is the purpose of your change?
I worked on the AP vs IB vs College classes

#### What areas of the application does this change impact? (changes API, folder structure, etc)
page

#### What features/fixes are deferred/not handled in this PR?
i believe everything was handled, i asked some questions about the format but i haven't gotten any feedback, i will be awaiting feedback if you need anything changed.

## Checklist
- [ ] Adds unit tests
- [ ] Has a work item linked to the PR

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,233 @@
...
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ul,
+ol,
+li {
```
Comment (michachan) [inline src/App.css]: As mentioned earlier, I can approve this now, but let's look for more scalable and consistent styling solutions. We'll need to refactor this vanilla CSS to another UI library- top ones to consider may be tailwind, material UI, chakra UI, and mantine

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,233 @@
...
+  height: 1020px;
+   padding: 20px;
+  flex-direction: column;
+  align-items: flex-start;
+  flex-shrink: 0;
+  border-radius: 20px;
+  background: #878787;
+  margin-top:49px;
+  margin-bottom: 48px;
+  margin-right: 46px; */
```
Comment (michachan) [inline src/App.css]: remove commented/unused code

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,153 @@
...
+                mean if I take college classes while I am still in high school?
+                These are all possible questions that you have had or are
+                currently wondering about. Especially with the looming idea of
+                wondering about your prospective university choices and what the
+                future may hold for you.
+              </p>
+              <h4 className="recipe_subHeader">Advanced Placement (AP)</h4>
+              <p className="recipe_text">
+                With AP classes that have an exam and course guidelines that
+                have been administered by the College Board. Which means by
```
Comment (michachan) [inline src/App.js]: nit, why not multiple <p> containers instead of double line breaks?

File: src/MainBox.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line prettier/prettier
+import React from "react";
+import PropTypes from "prop-types";
+
+export const MainBox = ({ children }) => {
+  // eslint-disable-next-line prettier/prettier
```
Comment (michachan) [inline src/MainBox.js]: why are these prettier disable rules here?

**Review by michachan — CHANGES_REQUESTED** [review]

**Comment by oo-ng** [comment]
I apologize for the late update, I was struggling with pushing the updates to github.

File: public/index.html [hunk 5]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>BoBaTalks</title>
```
Comment (michachan) [inline public/index.html]: check casing

**Review by michachan — APPROVED** [review]

**Pull request merged**